flink-table/flink-table-planner/src/main/java/org/apache/calcite/avatica/util/DateTimeUtils.java

```diff
@@ -882,6 +968,12 @@ public static int ymdToJulian(int year, int month, int day) {
 		int a = (14 - month) / 12;
 		int y = year + 4800 - a;
 		int m = month + 12 * a - 3;
+//		return day + (153 * m + 2) / 5
+//			+ 365 * y
+//			+ y / 4
+//			- y / 100
+//			+ y / 400
```

Comment:
Useless comments

Reply:
these lines are explicitly comment out on purpose.

Reply:
Let's just remove these useless comments, we can add them back if we really need them in the future, for current code base, these comments are really confusing.